framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/CopyText/index.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as React from 'react';
+import styles from './CopyText.module.scss';
+import Icon from '../Icon';
+import Text from '../Text';
+
+interface Props {
+	name: string;
+	size?: 's' | 'm' | 'l' | 'xl';
```

Comment:
these property i think you can export Props from Text and reference rather than redefining

Reply:
Yes it should be exported from `Text/index.tsx` and then union those types here.